### PR TITLE
Prep work for CloudQuery preview - CIS for AWS

### PR DESCRIPTION
**Pull request by jpipkin1**
## Purpose of this pull request

This pull request makes updates to this article in preparation for the CloudQuery preview:
https://help.sumologic.com/docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws/

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

## Ticket (if applicable)

NA. See [DOCS-70](https://sumologic.atlassian.net/browse/DOCS-70) for the preview work.

### CODE REVIEW

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md [hunk 1]
```diff
@@ -320,7 +320,9 @@ You can install Cloud Infrastructure Security for AWS from the App Catalog to us
               <details>
        <summary>CloudQuery</summary>
 
-       In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source/) in Sumo Logic. If fields are missing, or you need to change them, do the following:
+       In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source/) in Sumo Logic. (This functionality is in [preview](#cloudquery-preview). If would like to know more about the feature, reach out to your Sumo Logic Account Executive.)
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:323]:
```suggestion
       In this section, you have the choice to create a [CloudQuery source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/cloudquery-source) in Sumo Logic. (This functionality is in [preview](#cloudquery-preview). If you would like to know more about the feature, reach out to your Sumo Logic Account Executive.) 
```

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md [hunk 2]
```diff
@@ -500,6 +492,16 @@ This dashboard is in [preview](#cloudquery-preview). If you see only empty panel
...
 
+#### Security Control Failures Investigation
+
+The **Security Control Failures Investigation** dashboard provides a detailed view for the [**Security Control Failures Overview**](#security-control-failures-overview) dashboard and helps you navigate and prioritize the most important misconfigurations in your environment. 
+
+:::note
+This dashboard is in [preview](#cloudquery-preview). If you see only empty panels in the dashboard and would like to know more about the feature, reach out to your Sumo Logic Account Executive. 
+:::
+
+<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-security-control-failures-investigation.png')} alt="Security Control Failures Investigation dashboard" width="600"/>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:503]: should this have a gray border?

**Review by kimsauce — APPROVED** [review]
couple suggestions

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md (hunk 2 above)
Comment (jpipkin1) [reply docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:503]: Yes!

File: docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md [hunk 3]
```diff
@@ -400,17 +450,62 @@ The **Active Threats: AWS Resources** dashboard shows threats identified in AWS
...
+
+### Infrastructure Overview
+
+The **Infrastructure Overview** dashboard helps you identify all accounts, services, and resources within your cloud environment. It helps you get deep visibility into your cloud infrastructure to understand how many cloud resources are running and their configurations.
+
+:::note
+This dashboard is in [preview](#cloudquery-preview). If you see only empty panels in the dashboard and would like to know more about the feature, reach out to your Sumo Logic Account Executive. 
+:::
+
+<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-infrastructure-overview.png')} alt="Infrastructure Overview dashboard" width="600"/>
```
Comment (jpipkin1) [inline docs/integrations/amazon-aws/cloud-infrastructure-security-for-aws.md:473]:
```suggestion
<img src={useBaseUrl('img/integrations/amazon-aws/cis-for-aws-infrastructure-overview.png')} alt="Infrastructure Overview dashboard" style={{border: '1px solid gray'}} width="600"/>
```

**Pull request merged**